pkg/inventory/model/predicate.go

```diff
@@ -0,0 +1,407 @@
+package model
+
+import (
+	"bytes"
+	liberr "github.com/konveyor/controller/pkg/error"
+	"reflect"
+	"strings"
+	"text/template"
+)
+
+//
+// Label SQL.
+var LabelSQL = `
```

Comment:
Interpolating fields like this into a SQL query raises red flags for me. Is this sort of templating safe?

Reply:
Can you elaborate?

Reply:
Updated to use params to prevent SQL injection.